src/grappler/ngraph_optimizer.cc

```diff
@@ -66,18 +66,81 @@ Status NgraphOptimizer::Optimize(tensorflow::grappler::Cluster* cluster,
   if (config::IsEnabled() == false ||
       std::getenv("NGRAPH_TF_DISABLE") != nullptr) {
     NGRAPH_VLOG(0) << "Ngraph is disabled ";
+    graph.ToGraphDef(output);
     return Status::OK();
   }
 
   // Get the nodes to be skipped
   std::vector<string> fetch_nodes;
   for (const string& f : item.fetch) {
-    NGRAPH_VLOG(5) << "Skip fetch node: " << f;
     int pos = f.find(":");
     fetch_nodes.push_back(f.substr(0, pos));
```

Comment:
could we make `fetch_nodes` a set. As a vector it can potentially look like this:
<node1, node1, node2> (because node1 had 2 output tensors node1:0, node1:1)